src/plugins/meson.build

```diff
@@ -1,4 +1,5 @@
 incdir = include_directories('..', '../..')
+zlib_dep = dependency('zlib', required: true)
```

Comment:
I think this could be moved to be inside the `if` of the plugin that actually uses it